Mindbox.ExceptionsHandling.GraphQL/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using HotChocolate.Execution.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Mindbox.ExceptionsHandling.GraphQL;
+
+public static class ConfigurationExtensions
+{
+	public static IRequestExecutorBuilder AddGraphQLExceptionsHandling(this IRequestExecutorBuilder requestBuilder)
```

Comment:
тавтология, IRequestExecutorBuilder существует только в контексте GraphQL
```suggestion
	public static IRequestExecutorBuilder AddExceptionsHandling(this IRequestExecutorBuilder requestBuilder)
```